web/impact/impact/tests/test_analyze_judging_round_view.py

```diff
@@ -210,6 +210,18 @@ def test_get_with_commitment(self):
                     assert commitment.capacity - 1 == result[
                         'remaining_capacity']
 
+    def test_get_when_no_applications_for_round(self):
+        context = AnalyzeJudgingContext(add_application=False)
+        judging_round_id = context.criterion.judging_round.id
+        email = self.global_operations_manager(program_family).email
```

Comment:
`program_family` does not seem to be defined here. (Travis will probably complain about this)

Reply:
shoot, this didnt even error on me when i ran `make test`!! 😠